src/Persistence/Initialization/VersionSeasonSix/ChaosMixes.cs

```diff
@@ -853,11 +842,15 @@ private ItemCrafting DarkHorseCrafting()
         creation.MaximumAmount = 1;
         craftingSettings.RequiredItems.Add(creation);
 
+        // Result:
+        craftingSettings.ResultItemSkillChance = 100;
+
         var darkHorse = this.Context.CreateNew<ItemCraftingResultItem>();
         darkHorse.ItemDefinition = this.GameConfiguration.Items.First(i => i.Name == "Dark Horse");
         darkHorse.Durability = 255;
+        darkHorse.RandomMinimumLevel = 1;
+        darkHorse.RandomMaximumLevel = 1;
```

Comment:
https://github.com/kyleruss/emu-server/blob/202856a74c905c203b9b2795fd161f564ca8b257/GameServer/common/zzzitem.cpp#L77
Previously the Horse and Raven were coming out at level 0. I searched new and old forums and only saw level 1 Raven/Horse being sold.